packages/venia-concept/src/components/MiniCart/kebab.css

```diff
@@ -1,30 +1,24 @@
 .root {
-    position: relative;
     display: inline-block;
-
     grid-row-start: 1;
     grid-row-end: 4;
     grid-column-start: 3;
     grid-column-end: 4;
-    align-self: center;
-}
-
-.root ul {
-    z-index: 3;
-    position: absolute;
-    right: 0.5em;
-    box-shadow: 0 0 1px #999;
+    margin-top: 2px;
+    position: relative;
 }
 
 .dropdown {
+    align-items: center;
+    box-shadow: 0 0 1px rgb(0, 0, 0, 0.2);
     display: grid;
     position: absolute;
-    right: 0;
-    top: -2.5em;
-    transition: 250ms ease-out;
+    right: 2px;
+    top: 0;
```

Comment:
Align the dropdown to the top-right of the kebab. Otherwise the header overlaps this (separate stacking contexts and this one's ancestor is lower necessarily).